### PR TITLE
Feature/run script

### PR DESCRIPTION
**Pull request by GregSharpe1**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,11 @@
 # IAGv Infrastructure Readme File!
-This repository is a repository for all of IAGv infrastructure and configuration management. 
+ This repository is a repository for all of IAGv infrastructure and configuration management.
+
+# Dependancies
```
Comment (daviesnathan) [inline README.md]: Dependencies*

Comment (GregSharpe1) [inline README.md]: Cheers Nath, anything else you think should be in there?

Comment (daviesnathan) [inline README.md]: I wouldn't say so at this stage.

**Comment by GregSharpe1** [comment]
coolio cheese :+1: merging

**Pull request merged**